CONTRIBUTING.md

```diff
@@ -2,7 +2,10 @@
 
 Great that you are here and you want to contribute to n8n
 
-## Contents
+## Table of Contents
+
+<details>
```

Comment:
Let's not do this. Not much gained by hiding the TOC, and it forces another click for people who want to use it.